app/src/pages/Login/Login.tsx

```diff
@@ -0,0 +1,66 @@
+//pages/Login/Login
+
+import React, { useState } from 'react';
+import { TextInput } from '../../components/Input/TextInput/TextInput';
+import { Button } from '../../components/Button/Button';
+//import { useNavigate } from 'react-router-dom';
+
+export type LoginProps = {
+  handleLogin: (redirectPath: string) => void;
+};
+
+const Login = ({ handleLogin }: LoginProps) => {
+  const [userName, setUserName] = useState<string>('');
+  const [password, setPassword] = useState<string>('');
+  //const navigate = useNavigate();
+
+  const handleLoginClick = () => {
+    if (userName.trim() !== '' && password.trim() !== '') {
+      console.log('Login success');
+      localStorage.setItem('token', 'example_token');
+      // navigate('/dashboard');
+      handleLogin('/dashboard');
+    } else {
+      console.log('Login failed');
+    }
+  };
```

Comment:
Can this whole function be moved to the parent and handled inside of `handleLogin`?
atm when we click the login button in the story this whole function is triggered but shouldnt be,